client/main/instance.js

```diff
@@ -17,6 +17,8 @@ const ProgressModel = Backbone.Model.extend({
     }
 });
 
+
+
```

Comment:
try not to commit these empty changes ... it's good to review what's changed in each file before you commit. on windows and osx we have the graphical github client which makes this easy, but it's not available for linux (i don't think), you could try seeing what's available for linux.